managers/mapdb/src/main/kotlin/com/justai/jaicf/context/manager/mapdb/MapDbBotContextManager.kt

```diff
@@ -8,7 +8,14 @@ import com.justai.jaicf.context.manager.BotContextManager
 import org.mapdb.DBMaker
 import org.mapdb.Serializer
 
-class MapDbBotContextManager(dbFilePath: String? = null): BotContextManager {
+@Deprecated(
+    "MapDbBotContextManager is deprecated. Use JacksonMapDbBotContextManager instead",
+    ReplaceWith(
```

Comment:
Let's not add replace with here. If a user will decide to use this quick fix, he will break his db-file

Reply:
Also, let's add incompatibility notice in deprecation message